src/validations/rules/ssp.sch

```diff
@@ -2175,22 +2175,22 @@
             <!-- See List of Equipment and Services Covered By Section 2 of The Secure Networks Act - https://www.fcc.gov/supplychain/coveredlist  -->
             <sch:let 
                 name="prohibit-vendor" 
-                value="'Dahua Technology Company', 'Dahua',
+                value="('Dahua Technology Company', 'Dahua',
                 'Hangzhou Hikvision Digital Technology', 'Hangzhou', 
                 'Hikvision', 'Hangzhou Hikvision', 
                 'Huawei', 'Huawei Technologies Company', 
                 'HyTera', 'Hytera Communications Corporation', 
                 'AO Kaspersky Lab', 'Kaspersky Lab', 'Kaspersky', 
                 'ZTE', 'ZTE Corporation', 
-                'China Mobile International USA Inc',
-                'China Telecom (Americas) Corp'"/>
+                'China Mobile', 'China Mobile International USA Inc',
+                'China Telecom', 'China Telecom (Americas) Corp')"/>
             <sch:assert
```

Comment:
This assertion fails to exhibit a correct message for a postive test outcome.

The acceptance criterion is "**All Schematron assertion messages are declarative statements which affirm the positive test outcome.**".

The message you are proposing is "**The information system contains a banned vendor -  '<xsl:value-of select="o:prop[@name eq 'vendor-name']/@value"/>'.  See FAR 889(a)(1)(B)**.". When this assertion test - `not(o:prop[@name eq 'vendor-name']/@value = $prohibit-vendor)` - i.e., encounters an `<inventory-item>` `<prop name="vendor">` which is not a banned value, is positive (boolean true) for any `<inventory-item>` the Schematron result (SVRL output) will have the assertion message citing the value as banned when it is not.

When the assertion is successful, the message will be incorrect and additionally cite the correct vendor-name value as being a banned one.

A sample positive assertion message would be something like "_The inventory item does not cite a banned vendor._".

Let's discuss this tomorrow if questions remain.